tests/manage/mcg/test_pv_pool.py

```diff
@@ -146,56 +146,91 @@ def test_pv_scale_out(self, backingstore_factory):
+                                MIN_PV_BACKINGSTORE_SIZE_IN_GB,
+                                "ocs-storagecluster-ceph-rbd",
+                            )
+                        ]
+                    },
+                },
+                marks=[
+                    tier2,
+                    pytest.mark.polarion_id("OCS-3932"),
+                    pytest.mark.bugzilla("2064599"),
+                ],
+            ),
+            pytest.param(
+                {
+                    "interface": "CLI",
+                    "backingstore_dict": {
+                        "pv": [
+                            (
+                                3,
+                                MIN_PV_BACKINGSTORE_SIZE_IN_GB,
+                                "ocs-storagecluster-ceph-rbd",
+                                "300m",
+                                "500Mi",
+                                "400m",
+                                "600Mi",
+                            )
+                        ]
+                    },
+                },
+                marks=[tier2, pytest.mark.polarion_id("OCS-4643")],
+            ),
+        ],
+    )
+    def test_pvpool_resource_modifications(
         self,
         awscli_pod_session,
         backingstore_factory,
         bucket_factory,
         test_directory_setup,
         mcg_obj_session,
+        bucketclass_dict,
     ):
         """
-        Test to modify the CPU and Memory resource limits for BS and see if its reflecting
+        Objective of the test are:
+            1) See if the CLI options to add resource parameters works while creating
+            Pv based backingstore.
+            2) Modifying the backingstores resource, reflects in the pv based backingstore
+            pods.
+
         """
-        bucketclass_dict = {
-            "interface": "OC",
-            "backingstore_dict": {
-                "pv": [
-                    (
-                        1,
-                        MIN_PV_BACKINGSTORE_SIZE_IN_GB,
-                        "ocs-storagecluster-ceph-rbd",
-                    )
-                ]
-            },
-        }
         bucket = bucket_factory(1, "OC", bucketclass=bucketclass_dict)[0]
         bucket_name = bucket.name
         pv_backingstore = bucket.bucketclass.backingstores[0]
         pv_bs_name = pv_backingstore.name
         pv_pod_label = f"pool={pv_bs_name}"
-        pv_pod_info = get_pods_having_label(
+        pv_pod_obj = list()
+        for pod in get_pods_having_label(
             label=pv_pod_label, namespace=config.ENV_DATA["cluster_namespace"]
-        )[0]
-        pv_pod_obj = Pod(**pv_pod_info)
-        pv_pod_name = pv_pod_obj.name
-        logger.info(f"Pod created for PV Backingstore {pv_bs_name}: {pv_pod_name}")
-        new_cpu = "500m"
-        new_mem = "500Mi"
+        ):
+            pv_pod_obj.append(Pod(**pod))
+        req_cpu = "400m"
+        req_mem = "600Mi"
+        lim_cpu = "500m"
+        lim_mem = "700Mi"
```

Comment:
shouldn't we pass the values from outside the test function?

Reply:
since values used for testing is same for both the tests, we dont have to pass them outside the test as parameterize.